### PR TITLE
Fix timezone in jobs to display local time

### LINKED ISSUE #9: Fix Timezone in Jobs
# Overview
The system that tracks asynchronous jobs has a time zone problem. In this issue you'll fix that problem.

# Illustration of the problem
This animation shows the problem. We want the createdAt and updatedAt date/times to show in the local (Pacific) time zone. Instead, they show in UTC (the time at the Greenwich observatory in London, England, disregarding daylight savings time).

Image

# How to fix it
There are two separate problems causing the issue:

When we generate the time right now, we are generating a `LocalDateTime` which carries no time zone information with it
When we store the time in the database, we are storing it at as `LocalDateTime`, which carries no time zone information with it
So, there are three fixes needed:

First, change this in the file `HappyCowsApplication.java`

Change: `@EnableJpaAuditing`
To: `@EnableJpaAuditing(dateTimeProviderRef = "utcDateTimeProvider")`
Then add this method to the main class:
```java
    @Bean
    public DateTimeProvider utcDateTimeProvider() {
        return () -> {
          ZonedDateTime now = ZonedDateTime.now();
          return Optional.of(now);
        };
    }
```
Second, in the entity Job.java, change this:

OLD:
```java
   @CreatedDate
    private LocalDateTime createdAt;
    @LastModifiedDate
    private LocalDateTime updatedAt;
```
NEW:
```java
    @CreatedDate
    private ZonedDateTime createdAt;
    @LastModifiedDate
    private ZonedDateTime updatedAt;
```
Finally, change the jobsFixtures.js file, adding timezone to each date, basically appending `-800` to each one:

BEFORE:
```java
        "createdAt": "2022-11-13T19:49:58.097465",
        "updatedAt": "2022-11-13T19:49:59.203879",
```
AFTER:
```java
        "createdAt": "2022-11-13T19:49:58.097465-800",
        "updatedAt": "2022-11-13T19:49:59.203879-800",
```
# To test
Create a job as shown in the animation above.

The time shown should match the local time.

### PR DESCRIPTION
**Pull request by blakerandle**
- The `createdAt` and `updatedAt` times associated with jobs are now stored with timezone information.
- The `Created` and `Updated` columns in the jobs table now show the time in the local timezone, rather than UTC. 

Before:
image
After:
image

Closes #9

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
             <version>4.2.0</version>
-        </dependency>
+          </dependency>
```
Comment (ethwu) [inline pom.xml:106]: stray whitespace change

**Review by ethwu — APPROVED** [review]
lgtm

File: pom.xml (hunk 1 above)
Comment (blakerandle) [reply pom.xml:106]: That was not my change, that was Professor Conrad. He works in mysterious ways.

Comment (pconrad) [reply pom.xml:106]: lol

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**